dm/pkg/checker/mysql_server.go

```diff
@@ -35,14 +35,14 @@ func NewMySQLVersionChecker(db *sql.DB, dbinfo *dbutil.DBConfig) RealChecker {
 }
 
 // SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
-// * 5.6.0 <= MySQL Version < 8.0.0.
+// * 5.6.0 <= MySQL Version <= 8.0.24.
 var SupportedVersion = map[string]struct {
 	Min MySQLVersion
 	Max MySQLVersion
 }{
 	"mysql": {
 		MySQLVersion{5, 6, 0},
-		MySQLVersion{8, 0, 0},
+		MySQLVersion{8, 0, 24},
```

Comment:
I think we need more tests to officially support MySQL Version <= 8.0.24, let's make this change later

Reply:
Ok.  Updated in 6de0275701533b80fe6149e2e54d2173a1ac6a05